db/migrate/20200414143250_add_external_to_profiles.rb

```diff
@@ -1,6 +1,12 @@
 class AddExternalToProfiles < ActiveRecord::Migration[5.2]
-  def change
+  def up
     add_column :profiles, :external, :boolean, default: false, null: false
     add_index :profiles, :external
+    ExternalProfileUpdater.update!(DateTime.parse('2020-04-13T10:44:00+00:00'))
```

Comment:
Since we haven't deployed this migration yet, I think it makes sense to perform the data migration here.

Reply:
@dLobatog did you double check this date and time are correct?

Reply:
It looks right - the jenkins job finished at 12:45 minus 2h ~> 10:44

Reply:
You have to add a `require` to this migration - it's failing to load this class